testgrid/config.md

```diff
@@ -44,7 +44,7 @@ This functionality is provided by [Configurator](cmd/configurator).
 If you need to create a new dashboard, or do anything more advanced, read on.
 
 ## Configuration
-Open [`config.yaml`] in your favorite editor and:
+Open or create a [Testgrid config file][configuration] in your favorite editor and:
```

Comment:
I almost wonder if this `Configuration` section should be moved to `/config/testgrid/README.md` to talk about the mechanics of changing configuration for testgrid.k8s.io vs. testgrid itself

Reply:
It did occur to me that /config/testgrids/README.md should contain cluster-specific config information, while the /testgrid/* readmes should be more general, but that's tough to do because there's the cluster-specific tests still bundled in Configurator